src/frontend/apps/desk/src/features/mail-domains/components/panel/PanelItem.tsx

```diff
@@ -11,17 +12,31 @@ interface MailDomainProps {
 }
 
 export const PanelMailDomains = ({ mailDomain }: MailDomainProps) => {
+  const { t } = useTranslation();
   const { colorsTokens } = useCunninghamTheme();
   const {
     query: { slug },
   } = useRouter();
 
   const isActive = mailDomain.slug === slug;
 
+  const getStatusText = (status: MailDomain['status']) => {
+    switch (status) {
+      case 'pending':
+        return t('[pending]');
+      case 'enabled':
+        return t('[enabled]');
+      case 'disabled':
+        return t('[disabled]');
+      case 'failed':
+        return t('[failed]');
+    }
+  };
+
   const activeStyle = `
     border-right: 4px solid ${colorsTokens()['primary-600']};
     background: ${colorsTokens()['primary-400']};
-    span{
+    span:not(:last-child){
```

Comment:
I am not sure it is what you wanted to do, you loose the color effect on all your texts.

Reply:
Indeed, I intended to use different colors for the mail domain status according to the status value. But we will eventually keep it grey. I bring back the former CSS rule